scaladoc/resources/dotty_res/scripts/ux.js

```diff
@@ -285,7 +285,7 @@ document
         getTocListElement(id).parentElement.classList.toggle("active");
       }
       if (lastEntry.isIntersecting) {
-        window.location.hash = "";
+        history.replaceState(null, "", window.location.pathname + window.location.search);
```

Comment:
Let's preserve the state:
```suggestion
        history.replaceState(history.state, "", window.location.pathname + window.location.search);
```

Reply:
Done, thank you